start_server.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [ -f sam.pid ]; then
+  ./stop_server.sh
+fi
+sam local start-lambda --parameter-overrides 'ParameterKey=RuntimeLayer,ParameterValue='"$1"' ParameterKey=RecommendedLayer,ParameterValue='"$2"' ParameterKey=AWSLayer,ParameterValue='"$3"'' &
```

Comment:
Have you checked if there is a SAM-local python API? Going through bashes is quite ugly.